laws/src/main/scala/cats/laws/UnorderedTraverseLaws.scala

```diff
@@ -0,0 +1,62 @@
+package cats
+package laws
+
+import cats.data.Nested
+
+trait UnorderedTraverseLaws[F[_]] extends UnorderedFoldableLaws[F] {
```

Comment:
This could/should extend `FunctorLaws[F]` and assert some things about the compatibility of `UnorderedTraverse` and `Functor`; for instance, I would expect `unorderedSequence` to  be `unorderedTraverse` on identity (which assumes that `Eq` for these instances respects unorderedness if it's not also a `Traverse`, I think)

Reply:
Hmmm, I'm not sure. That would imply that `Set` would be a valid `Functor` which I believe it is not. We could add a function like `unorderedMap` that's just `unorderedTraverse` with `Id`.

Thanks for your review by the way, it's been very helpful! :)

Reply:
No, you're right; I got a little bit overenthusiastic about drawing these parallels. 

I think `unorderedMap` is kinda dubious, because the difference isn't even lack of order (which is unobservable in `Functor#map` already!), but that the operation can "drop" values, losing fusion, and therefore isn't a (categorical) `Functor`. I don't know what laws for `unorderedMap` would be, other than "it's the same as traversing with `Id`.